src/scss/dp/overrides/components/_release-calendar.scss

```
@@ -11,4 +11,14 @@
       }
     }
   }
+  .ons-pagination {
+    @media screen and (max-width: 980px) {
+      &__item {
+        &:not(:where(.ons-pagination__item--previous, .ons-pagination__item--next)){
```

Add a space between closing parenthesis and opening brace

```suggestion
        &:not(:where(.ons-pagination__item--previous, .ons-pagination__item--next)) {
```